### PR TITLE
added anat type matlab code and template

### PR DESCRIPTION
**Pull request by anushkab**
I have added the matlab file and a json template for anat data type.
Resolved issue https://github.com/INCF/bids-starter-kit/issues/21

### CODE REVIEW

File: matlabCode/createBIDS_anat_T1w_json.m [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+              '_acq-' acq_id ...
+              '_run-' run_id '_T1w.json']);
+
+% Assign the fields in the Matlab structure that can be saved as a json.
+%The following fields must be defined:
+
+%%
+%The “effective” sampling interval, specified in seconds, between lines in the phase-encoding direction, defined based on the size of the reconstructed image in the phase direction.  It is frequently, but incorrectly, referred to as  “dwell time” (see DwellTime parameter below for actual dwell time).  It is  required for unwarping distortions using field maps. Note that beyond just in-plane acceleration, a variety of other manipulations to the phase encoding need to be accounted for properly, including partial fourier, phase oversampling, phase resolution, phase field-of-view and interpolation. This parameter is REQUIRED if corresponding fieldmap data is present. 
+
+anat_json.EffectiveEchoSpacing='0.0000074';
```
Comment (dorahermes) [inline matlabCode/createBIDS_anat_T1w_json.m]: There are many missing fields. There are two types of templates we are hoping to provide, one with all required/recommended/optional fields and one with only required fields. There seem to be many more fields than just these two.

**Comment by dorahermes** [comment]
Would it be possible to adjust such that either all requires/recommended/optional fields are there OR only the required ones in the template (FULL template versus SHORT template)? Thank you!!

**Comment by anushkab** [comment]
I have now created two matlab codes for generating Full(with all required/recommended /optional fields) and Short (required fields only)Template.
I have also updated the respective output json files.

Regards
Anushka

**Comment by dorahermes** [comment]
lgtm, thank you!

**Pull request merged**